CodeStarterCamp_Week4/main.swift

```diff
@@ -1,12 +1,82 @@
 //
-//  main.swift
+//  week4_step1.swift
 //  CodeStarterCamp_Week4
 //
-//  Created by yagom.
-//  Copyright © yagom academy. All rights reserved.
+//  Created by 정경우 on 2022/11/17.
 //
 
-import Foundation
+class BodyCondition {
+    var upperStrength: Int = 0 {
+        didSet(oldValue) {
+            print("상체근력이 \(upperStrength - oldValue) 상승합니다.")
+        }
+    }
+    var lowerStrength: Int = 0 {
+        didSet(oldValue) {
+            print("하체근력이 \(lowerStrength - oldValue) 상승합니다.")
+        }
+    }
```

Comment:
```suggestion
    var upperStrength: Int = 0 {
        didSet(oldValue) {
            print("상체근력이 \(upperStrength - oldValue) 상승합니다.")
        }
    }
    
    var lowerStrength: Int = 0 {
        didSet(oldValue) {
            print("하체근력이 \(lowerStrength - oldValue) 상승합니다.")
        }
    }
```
개행이 되어 있는 곳도 있고 안된 곳도 있는데 통일해주세요!

Reply:
수정했습니다